CMakeLists.txt

```diff
@@ -39,7 +39,9 @@ link_directories(
 set (plugins_single_header
         ArduPilotPlugin
         ArduCopterIRLockPlugin
+	ArduCopterPlugin
```

Comment:
That would make some problem. ArduPilotPlugin handles almost everything but,
Iris's rotor did not move using ArduPilotPlugin.